extensions/amp-access-laterpay/amp-access-laterpay.md

```diff
@@ -0,0 +1,17 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!-- TODO: describe -->
```

Comment:
Can you remove this file until there's actually content in it because it's getting pulled into the amproject docs site and is appearing empty - https://www.ampproject.org/docs/reference/components/amp-access-laterpay.

Alternatively, you can append "internal" to the filename and it will get excluded from being pulled into the docs build.